src/main/java/Storage.java

```diff
@@ -0,0 +1,121 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    protected ArrayList<Task> arr;
+    protected String filePath;
+
+    /**
+     * Constructor to initialize Storage class.
+     * @param filePath
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        this.arr = new ArrayList<Task>();
+    }
+
+    /**
+     * Method to begin the file reading process from history.
+     * @return
+     * @throws IOException
+     */
+    public ArrayList<Task> load() throws IOException {
```

Comment:
For the load method, refer to the comment on the parse method from the Parse class. Making your method modular is also applicable here.